packages/@react-spectrum/s2/src/TableView.tsx

```diff
@@ -792,7 +792,7 @@ function ColumnWithMenu(props: ColumnWithMenuProps) {
         </Button>
         <Menu onAction={onMenuSelect} styles={style({minWidth: 128})}>
           {items.length > 0 && (
-            <MenuSection aria-label={stringFormatter.format('table.standardColumnMenu')}>
+            <MenuSection>
```

Comment:
no label because we don't use the `h` tag anymore?

Reply:
we are missing translations for it and unlikely to get them before release. but regardless, in the example this is the only section that has a label, and I don't think it's really necessary. It's kind of an implementation detail that they are the "default actions"? I don't think the user really needs to know that.